### PR TITLE
Asset deprecation

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
Corrections from today's meeting.

### CODE REVIEW

**Review by mcampo — APPROVED** [review]
👍

**Comment by wwarner-salesforce** [comment]
@kuntald if there are no objections I'll merge this today.

File: modules/ROOT/pages/to-deprecate-asset.adoc [hunk 1]
```diff
@@ -6,8 +6,8 @@ endif::[]
 Deprecating removes access to a version of an asset. The asset remains accessible for users.
 A deprecated asset can be reverted and returned to use.
 
-You must be an organization owner or an Exchange Administrator to use this feature. You can set these roles
-in the Access Management > Roles menu.
+You must be an asset administrator, an organization owner or an Exchange Administrator to use
```
Comment (dukesphere) [inline modules/ROOT/pages/to-deprecate-asset.adoc]: should "Administrator" for Exchange be upper case vs "asset administrator," where it's lower

**Review by dukesphere — APPROVED** [review]
I'll approve and let you decide whether the small change is needed or not

File: modules/ROOT/pages/to-deprecate-asset.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/to-deprecate-asset.adoc]: It should be "administrator". Fixed. Thank you!

**Pull request merged**